### PR TITLE
feat(llc): add `Filter.contains` and `Filter.empty`

### PR DESCRIPTION
**Pull request by imtoori**

### CODE REVIEW

File: packages/stream_chat/lib/src/core/models/filter.dart [hunk 1]
```diff
@@ -157,13 +161,20 @@ class Filter extends Equatable {
...
+
   /// Creates a custom [Filter] if there isn't one already available.
   const factory Filter.custom({
     required Object value,
     String? operator,
     String? key,
   }) = Filter.__;
 
+  /// An empty filter
+  factory Filter.empty() => const Filter.raw(value: {});
```
Comment (xsahil03x) [inline packages/stream_chat/lib/src/core/models/filter.dart:176]: const factory maybe

Comment (imtoori) [reply packages/stream_chat/lib/src/core/models/filter.dart:176]: Only redirecting factory constructors can be declared to be `const`

File: docusaurus/docs/Flutter/guides/understanding_filters.mdx [hunk 2]
```diff
@@ -107,6 +107,36 @@ The 'exists' filter matches values that exist, or don't exist, based on the spec
...
+```dart
+Filter.raw(value: {
+  'members': [
+    ..._selectedUsers.map((e) => e.id),
+    chatState.currentUser!.id,
+  ],
+  'distinct': true,
+});
+```
+
```
Comment (xsahil03x) [inline docusaurus/docs/Flutter/guides/understanding_filters.mdx:139]: can you also add example for `Filter.custom`?

Comment (imtoori) [reply docusaurus/docs/Flutter/guides/understanding_filters.mdx:139]: done

**Review by xsahil03x — APPROVED** [review]
LGTM

**Pull request merged**